chia/plotting/prover.py

```diff
@@ -27,8 +29,13 @@ def get_compression_level(self) -> uint8: ...
     def get_version(self) -> PlotVersion: ...
     def __bytes__(self) -> bytes: ...
     def get_id(self) -> bytes32: ...
-    def get_qualities_for_challenge(self, challenge: bytes32) -> list[bytes32]: ...
-    def get_partial_proofs_for_challenge(self, challenge: bytes32) -> list[bytes]: ...
+    def get_qualities_for_challenge(self, challenge: bytes32, plot_strength: uint8) -> list[bytes32]: ...
```

Comment:
do we need this not that we have get_partial_proofs_for_challenge, we said we get the quality in v2 from the partial proof, i think get_qualities_for_challenge is redundant or at least we need some way to know which quality belongs to which partial